read-fonts/src/tables/glyf.rs

```diff
@@ -711,6 +728,7 @@ struct Contour<'a> {
     points: &'a [Point<F26Dot6>],
     /// The flags for points, sliced from a glyphs flag stream
     flags: &'a [PointFlags],
+    offcurve_first_mode: ToPathStyle,
```

Comment:
This name is used in lots of places but doesn't really match the type. Nothing seems particularly good, but maybe just `path_style` (to avoid the ugly "to" prefix everywhere)?

Reply:
Will fix.